okhttp/src/main/java/okhttp3/internal/http/RetryAndFollowUpInterceptor.java

```diff
@@ -128,7 +128,7 @@ public StreamAllocation streamAllocation() {
       } catch (RouteException e) {
         // The attempt to connect via a route failed. The request will not have been sent.
         if (!recover(e.getLastConnectException(), streamAllocation, false, request)) {
-          throw e.getLastConnectException();
```

Comment:
Is this now unused?

Reply:
No, used 1 line above in determining whether error is recoverable.

Reply:
Well I suppose my search wasn't exhaustive!